osf/migrations/0129_create_cache_table.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0128_merge_20180829_0012'),
+    ]
+
+    operations = [
+        migrations.RunSQL([
```

Comment:
Is this migration necessary? There's a command for this: `python manage.py createcachetable`. Will Django fail gracefully if the database table doesn't exist? If so, then I think this can be removed.

Reply:
Django doesn't fail gracefully essentially it acts like there's an unrun migration.

Reply:
>  it acts like there's an unrun migration.

Does that mean that 1) the app doesn't start or 2) an error gets raised when calling `cache.get`? If it's the former, that's good because the app containers won't fail over, which will give us a chance to run the `createcachetable` command.

Reply:
It fails when you call `cache.get`. Devs will get an opportunity to run `createcachetable`

Reply:
Ah, ok. Then I think this migration can stay. Just one suggestion: it should read from `settings` to get the table name. That way, if that setting changes, this migration won't create the wrong table.